joomla/joomla-cms
# TinyMCE support for inline multi-language tags

Pull Request for Issue https://github.com/joomla-projects/accessibility/issues/2
a redo of https://github.com/joomla/joomla-cms/pull/11366
#### Summary of Changes

Support for `<span lang="en" dir="ltr" xml:lang="en"` depending on the language selected on the dropdown button.
#### Testing Instructions

Apply patch.
Make sure that you have more than one language installed in joomla, either wise you won't see something new :)

Select some text and then select the required language from the language button (it's a drop down)

Inspect the selected text to make sure that the text is wrapped with the span and the language tag.

Toggle the editor to inspect the actual html code, re toggle few more times to ensure that changes persist.

Save and see if changes are retained. ~~Right now this seems to fail, it must be some weird php side filtering.~~
#### Preview

<img width="1117" alt="screen shot 2016-07-30 at 17 47 07" src="https://cloud.githubusercontent.com/assets/3889375/17271182/e7d32158-567d-11e6-9474-218cb3d2b959.png">
